rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -0,0 +1,145 @@
+from typing import Any, Dict, List, Text
+
+from rasa.nlu.tokenizers.tokenizer import Token
+from rasa.nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.training_data import Message, TrainingData
+from rasa.nlu.constants import (
+    CLS_TOKEN,
+    MESSAGE_ATTRIBUTES,
+    TOKENS_NAMES,
+    TEXT_ATTRIBUTE,
+)
+import tensorflow as tf
+
+
+class ConveRTTokenizer(WhitespaceTokenizer):
+
+    provides = [TOKENS_NAMES[attribute] for attribute in MESSAGE_ATTRIBUTES]
+
+    defaults = {
+        # Flag to check whether to split intents
+        "intent_tokenization_flag": False,
```

Comment:
I think other tokenizersdon't provide this functionality, and users would need to add whitespace one for spacy. I think, it makes sense to silent this behavior here

Reply:
the whitespace tokenizer has the same options